railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -12,10 +12,10 @@
     <%- else -%>
       <%- if gemfile_entries.any? { |m| m.name == 'turbolinks' } -%>
     <%%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
```

Comment:
Actually, it looks like this is coming down the pipeline. [`link` tags will soon support `async`, non-render-blocking behavior with the `import` behavior](https://w3c.github.io/webcomponents/spec/imports/#link-type-import). You can use this in Chrome and Opera today, but Firefox has no plans to support it and IE doesn't look it does either.